### DISCUSSION TITLE
Regex Queries: Is there a way to only search in a specific folder of feeds?

### DISCUSSION THREAD

**Discussion by aleyvo** [martinrotter/rssguard#1239] [Q&A]
Is there a way for regex queries to only search for articles from a **specific folder of feeds**?
I do dozens of searches every day in RSS Guard, always typing the same terms, but in specific folders.
Couldn't find a way to do this through Regex Queries.

BTW, thank you @martinrotter, for this amazing piece of software. Have been using it for almost two years now. Keep up the good work!

**Comment by martinrotter** [comment]
Well, at this point regex queries run on all items from "account" they belong to.

I guess the feature you request could be added in the future. If you still want to, create feature request in "issues" section. :)
